README.md

```diff
@@ -606,29 +670,56 @@
     }
     ```
 
-    **[[⬆]](#TOC)**
+  - Prefixing your comments with `FIXME` or `TODO` helps other developers quickly understand if you're pointing out a problem that needs to be revisited, or if you're suggesting a solution to the problem that needs to be implemented. These are different than regular comments because they are actionable. The actions are `FIXME -- need to figure this out` or `TODO -- need to implement`.
```

Comment:
It wasn't exactly intentional, but either will work with most editor tools.  Did you have a strict preference for one over the other?